examples/complete/README.md

```diff
@@ -24,8 +24,9 @@ Ensure that you have the following tools installed locally:
 To provision this example:
 
 ```sh
+aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
```

Comment:
I believe that if we add the following to the example, this should resolve this without any additional CLI commands for users:

```hcl
data "aws_ecr_authorization_token" "token" {}
```

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ecr_authorization_token

Reply:
added

Reply:
were you able to test if this works as intended?